types/spec/signal.d.ts

```diff
@@ -12,7 +12,7 @@ export interface PushSignal extends BaseSignal {
   push?: 'outer';
 }
 export interface NewSignal extends BaseSignal {
-  value?: string;
+  value?: any;
```

Comment:
Why do we need any here? Can we find a better type?

Reply:
The [signal documentation](https://vega.github.io/vega/docs/signals/) says the value type is Any. Also the [return value type of View.signal is any](https://github.com/vega/vega-typings/blob/master/types/runtime/index.d.ts#L57). 

How about this: I can introduce a SignalValue type - which is a start towards defining all possible signal types. It will be **any** initially, but then we can add more known types.

Reply:
Yeah, that sounds like a good idea. Let's do that in a follow up.